docs/gettsim_objects/input_variables.rst

```diff
@@ -146,3 +140,15 @@ household.
 || _`y_pflichtbeitr_ab_40`  || Total years of mandat. contributions after || float       |
 ||                          || age 40                                     ||             |
 +---------------------------+---------------------------------------------+--------------+
+|| _`anwartschaftszeit`     || Months of unemployment insurance           || int         |
+||                          || contributions in the last 30 months        ||             |
```

Comment:
should be more precisely "in the 30 months before claimining unemployment insurance" if I am not mistaken